my_tests/dirk_step.jl

```diff
@@ -0,0 +1,206 @@
+using Pkg
```

Comment:
Please try to stick to gridap code style guide, e.g., the way we name files here is not following it

Reply:
removed in clean version